core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v1/LdapConfigEndpoint.java

```diff
@@ -104,4 +105,11 @@
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = LdapConfigOpDescription.POST_CONNECTION_TEST, produces = ContentType.JSON, nickname = "postLdapConnectionTest")
     LdapTestResult testLdapConnection(@Valid LDAPTestRequest ldapValidationRequest);
+
+    @GET
+    @Path("{name}/request")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = OperationDescriptions.LdapConfigOpDescription.GET_REQUEST, produces = ContentType.JSON, notes = Notes.LDAP_CONFIG_NOTES,
+            nickname = "getLdapRequestFromName")
+    LdapConfigRequest getRequestfromName(@PathParam("name") String name);
```

Comment:
`getRequestfromName` isn't in well formatted. `getRequestFromName` would be better (everywhere)